plugins/home/src/homePageComponents/Toolkit/Content.tsx

```diff
@@ -35,8 +35,12 @@ const useStyles = makeStyles(theme => ({
   },
   label: {
     marginTop: theme.spacing(1),
+    minWidth: '64px',
```

Comment:
Should we just set `width` instead?

Reply:
Thanks again for the review; in hindsight, a strict definition is definitely better for uniformity than a window of expansion. Set width to '72px' because I couldn't fit the word 'Automation' (example from issue) on one line without pushing one character onto the next one at '64px'